### PR TITLE
Upgrade stylelint

### PR DESCRIPTION
**Pull request by antleblanc**
⚠️ Please do not merge ⚠️ 

### Description of the Change

9fd3f84 — build(deps): upgrade stylelint to v9.9.0
ff52988 — style: apply stylelint configuration

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-manager#598.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✘  
  * Build manager EU ✘  
  * Build manager US ✘  
  * Tests ✔

File: client/app/css/less/common/informationBlock.less [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
   border: 1px solid;
-  .background-color-fn(@barBlockBackground);
-  .color-fn(@barBlockFont);
-  .border-color-fn(@barBlockBorder);
-  .rounded(@RADIUS_CONST);
+  .background-color-fn(@barblockbackground);
+  .color-fn(@barblockfont);
+  .border-color-fn(@barblockborder);
+  .rounded(@radius_const);
   // .box-shadow( 2px 2px 2px @grey3);
```
Comment (marie-j) [inline client/app/css/less/common/informationBlock.less]:
```suggestion

```
?

Comment (antleblanc) [reply client/app/css/less/common/informationBlock.less]: Good catch. Now removed.

**Comment by antleblanc** [comment]
I close this pull request because some styles are broken due to update on variable references. I will dig into it later in order to reopen it.

**Pull request closed without merging**